docs/software/file_servers.md

```diff
@@ -2,13 +2,171 @@
 
 ## Overview
 
-!!! warning "Documentation in progress..."
+- [**ProFTPD - Simple, efficient, lightweight FTP file server**](#proftpd-simple-efficient-lightweight-ftp-file-server)  
+- [**Samba - Feature rich file server**](#samba-feature-rich-file-server)  
+- [**vsftpd - Feature rich FTP file server**](#vsftpd-feature-rich-ftp-file-server)  
+- [**NFS - Network file system server**](#nfs-network-file-system-server)  
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
```

Comment:
Do we want to skip that `dietpi-launcher` step? I mean it is nice to have an overview of all DietPi tools but in a step-by-step guide lets reduce the required steps and advice to call `dietpi-software` directly.

Reply:
I also prefer only to describe `dietpi-software`.  
I will add this to the todo list as a global change.